src/api/PDFDocument.ts

```diff
@@ -1264,6 +1265,20 @@ export default class PDFDocument {
     return dataUri ? `data:application/pdf;base64,${base64}` : base64;
   }
 
+  findPageForRef(ref: PDFRef): PDFPage {
```

Comment:
Can we rename this so that it's clear the ref should be that of an annotation?
```suggestion
  findPageForAnnotationRef(ref: PDFRef): PDFPage {
```